### DISCUSSION TITLE
Flybywire A320 Neo

### DISCUSSION THREAD

**Discussion by ow1962** [maartentamboer/X-Touch-Mini-FS2020#63] [Q&A]
Hi Maarten:

I love your project. It is so clean and responsive.  Bravo

One problem that has come up since flybywire updated the autopilot and engine modules on the a32NX, is that the A:variables have changed so that it no longer responds to the standard commands for speed, heading and vs. It also doesnt respond to push or pulls anymore. Do you know how we can modify the events to recognize this aircraft.  The larest version of the plane is really good and I love the idea of using my X-touch

Any help would be appreciated

**Comment by maartentamboer** [comment]
For everything that goes through the Mobiflight wasm module you can change the events.txt to bind to new variables.
Regarding all the events that are directly in the configuration, it is up to the FBW team to give us the ability to call an event to change things like heading and speed. Maybe contact them in their Discord to see if there are new events defined

**Comment by JLD95** [comment]
Hello, I would like to use XTouch with last FBW A320Neo in particular to modify local variables. If I install Mobiflight, I can with X-Touch-Mini-FS2020 to read/write A320NX local variables ? Can you provide an example ?
At now  "simvar_led": "SPOILERS_ARMED" don't work anymore because the new variable is A32NX_SPOILERS_ARMED

**Reply by elewarr** [reply]
It may be possible to at least write variables. Please check CRJ configuration to see how it's done with user defined variables:

- definitions: https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/Configurations/Functions/CRJ/events.user.txt
- usage: https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/Configurations/config_crj.json

For A320 you may also have to create your own definitions if MobiFlight doesn't yet have them.

**Reply by JLD95** [reply]
OK thank you I will see the CRJ configuration.
To have Mobiflight fonctionnaly, it is necessary to install "MobiFlightConnector-8.0.2" or it is possible to install only standalone Mobiflight wasm module ? I don't see where I can find it

**Comment by ow1962** [comment]
Hi Maarten:

Here are some of the new A320NX events. I have tried some and they work if they are added to msfs2020.events.cip and events.txt

A32NX.FCU_AP_1_PUSH
A32NX.FCU_AP_2_PUSH
A32NX.FCU_SPD_PUSH
A32NX.FCU_SPD_PULL
A32NX.FCU_SPD_MACH_TOGGLE_PUSH
A32NX.FCU_HDG_PUSH
A32NX.FCU_HDG_PULL
A32NX.FCU_TRK_FPA_TOGGLE_PUSH
A32NX.FCU_ALT_PUSH
A32NX.FCU_ALT_PULL
A32NX.FCU_VS_PUSH
A32NX.FCU_VS_PULL
A32NX.FCU_LOC_PUSH
A32NX.FCU_APPR_PUSH
A32NX.FCU_EXPED_PUSH

I have a question though. How do I assign layer B functions

Thanks

Andrew


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by JLD95** [reply]
Hi, where is the file "msfs2020.events.cip" ? not in folder mobiflight-event-module and not in folder X-Touch-Mini-FS2020

**Comment by bigbrowncow** [comment]
This might help https://github.com/maartentamboer/X-Touch-Mini-FS2020/discussions/89
